src/bookshelf/rebar.lock

```diff
@@ -34,9 +34,10 @@
        {ref,"131e660ee39254a58b75075e07dfd742f445bfce"}},
   0},
  {<<"erlware_commons">>,
-  {git,"git://github.com/chef/erlware_commons.git",
+  {git,"https://github.com/chef/erlware_commons.git",
```

Comment:
any specific reason for this change?

Reply:
I noticed that the `rebar3 upgrade` procedure made this change in a few cases.  I have no idea why.